CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 This file contains a list of changes, new features, and fixes in each release of Monite SDK.
 
+## @monite/sdk-react@3.13.0 & @monite/sdk-api@3.13.0
+This package is a minor release with no breaking changes.
+
+### New features and improvements
```

Comment:
I see some more features that are worth no mention, like
- handling errors for invoice preview
- fix ReceivableTable component export
- added quick actions for the ReceivableTable component
- added sorting to the ReceivableTable component
- added the Due date for the InvoiceTable component

could you add these items as well?

Reply:
thanks!